Libraries/StyleSheet/__tests__/processTransform-test.js

```diff
@@ -46,33 +56,51 @@ describe('processTransform', () => {
       expect(() =>
         processTransform([{translate: 10}]),
       ).toThrowErrorMatchingSnapshot();
+      expect(() =>
+        processTransform('translate(10)'),
+      ).toThrowErrorMatchingSnapshot();
     });
 
     it('should accept a valid matrix', () => {
       processTransform([{matrix: [1, 1, 1, 1, 1, 1, 1, 1, 1]}]);
+      processTransform('matrix(1, 1, 1, 1, 1, 1, 1, 1, 1)');
       processTransform([
         {matrix: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]},
       ]);
+      processTransform(
+        'matrix(1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1)',
+      );
     });
 
     it('should throw when passing a matrix of the wrong size', () => {
       expect(() =>
         processTransform([{matrix: [1, 1, 1, 1]}]),
       ).toThrowErrorMatchingSnapshot();
+      expect(() =>
+        processTransform('matrix(1, 1, 1, 1)'),
+      ).toThrowErrorMatchingSnapshot();
     });
 
     it('should accept a valid translate', () => {
       processTransform([{translate: [1, 1]}]);
+      processTransform('translate(1, 1)');
       processTransform([{translate: [1, 1, 1]}]);
+      processTransform('translate(1, 1, 1)');
     });
 
     it('should throw when passing a translate of the wrong size', () => {
       expect(() =>
         processTransform([{translate: [1]}]),
       ).toThrowErrorMatchingSnapshot();
+      expect(() =>
+        processTransform('translate(1)'),
```

Comment:
This is valid (single length). Non-0 number is still invalid as length though

Reply:
I've just updated `processTransform` to support translate with only 1 argument as well